test/test_run_detext.py

```diff
@@ -131,3 +131,10 @@ def test_run_detext_multitask(self):
         sys.argv[1:] = args
         main(sys.argv)
         self._cleanUp(output)
+
+
+def test_demo():
```

Comment:
Is there missing indent? This should be included into the class method?

Reply:
it's a top-level test, as it's not testing the python file directly.